src/elements/content-explorer/ContentExplorer.js

```diff
@@ -796,16 +796,17 @@ class ContentExplorer extends Component<Props, State> {
      */
     attemptThumbnailGeneration = async (item: BoxItem): Promise<void> => {
         const { features } = this.props;
-        const { representations } = item;
-        const representation = getProp(representations, 'entries[0]');
+        const entries = getProp(item, 'representations.entries');
         const fileAPI = this.api.getFileAPI(false);
 
-        if (isFeatureEnabled(features, 'contentExplorer.gridView.enabled')) {
-            const updatedRepresentation = await fileAPI.generateRepresentation(representation);
+        if (isFeatureEnabled(features, 'contentExplorer.gridView.enabled') && entries) {
+            const updatedRepresentation = await fileAPI.generateRepresentation(entries[0]);
```

Comment:
It seems like doing the truthy check for `updatedRepresentation` is unnecessary since it always returns at least the same representation that was passed in. What we may want to to though is to consider doing an equality check to avoid an unnecessary `setState` call if nothing has changed. That or change `generateRepresentation` to return null in case the generation failed